test/browser/features/trace-propagation.feature

```diff
@@ -1,78 +1,84 @@
 Feature: Trace propagation headers
 
-    Scenario: traceparent header is added to xhr requests
-        Given I navigate to the test URL "/traceparent"
+    Scenario: traceparent and tracestate header is added to xhr requests
+        Given I navigate to the test URL "/trace-propagation"
         When I click the element "xhr"
 
         And I wait to receive a reflection
         Then the reflection request method equals "GET"
         And the reflection "X-Test-Header" header equals "test"
         And the reflection "traceparent" header matches the regex "^00-[A-Fa-f0-9]{32}-[A-Fa-f0-9]{16}-01"
+        And the reflection "tracestate" header matches the regex "^sb=v:1;r:[0-9]{0,32}"
```

Comment:
I don't think this should ever be 0 characters — wouldn't it always be at least 1?